app/api.js

```diff
@@ -1,6 +1,19 @@
 import { arrayToB64, b64ToArray, delay } from './utils';
 import { ECE_RECORD_SIZE } from './ece';
 
+let fileProtocolWSSUrl =
+  (localStorage && localStorage.getItem('wssURL')) ||
```

Comment:
`localStorage` here breaks the service worker. a try/catch should fix it

Reply:
thanks.